spec/controllers/application_controller/explorer_spec.rb

```diff
@@ -40,7 +40,8 @@
         ems = FactoryGirl.create(:ems_vmware, :ems_folders => [ems_folder])
 
         user = FactoryGirl.create(:user_admin)
-        user.current_group.set_managed_filters([["/managed/service_level/gold"]])
+        user.current_group.entitlement = Entitlement.create!(:filters => {'managed'   => [["/managed/service_level/gold"]],
+                                                                          'belongsto' => []})
```

Comment:
Well, actually I guess I'd prefer to leave them. Many specs previously just set `filters=` and I did this to avoid having to add endless lines of code that shouldn't be accessed that way in the first place, and the point here is moving rather than changing.